changelog.md

```diff
@@ -9,6 +9,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Misc and Bugfixes
 
 ## 0.8.3 (unreleased)
+### Breaking Changes
```

Comment:
Everything looks good to me @mikemhenry ! Just one thing, since this is a breaking change, I think it would be best to warn users whenever they use this method for the next few months or more. 

If you feel comfortable adding a `breaking_change` warning decorator in [mbuild.utils.decorators](https://github.com/mosdef-hub/mbuild/blob/master/mbuild/utils/decorators.py)
That would be excellent. We can include it in this PR if you want.

Reply:
I will add that to the PR, thanks!

Reply:
Okay looks like it is working in the CI `/home/travis/build/mosdef-hub/mbuild/mbuild/utils/decorators.py:21: Warning: write_hoomdxml has breaking change. See PR#463 on github
  warn(printed_message, Warning)` Will we remove this after a few versions?